xarray/backends/api.py

```diff
@@ -1393,6 +1394,7 @@ def open_mfdataset(
     join: JoinOptions = "outer",
     attrs_file: str | os.PathLike | None = None,
     combine_attrs: CombineAttrsOptions = "override",
+    errors: str = "raise",
```

Comment:
```suggestion
    errors: ErrorOptionsWithWarn = "raise",
```
From `xarray.core.types`

Reply:
Thanks @headtr1ck I did not know about `xarray.core.type`.